rqd/rqd/rqcore.py

```diff
@@ -402,6 +413,13 @@ def runWindows(self):
                                                           self.rqCore.updateRss)
             self.rqCore.updateRssThread.start()
 
+        while True:
+            output = frameInfo.forkedCommand.stdout.readline()
+            if not output and frameInfo.forkedCommand.poll() is not None:
+                break
+            if output:
+                self.rqlog.write(output, prependTimestamp=rqd.rqconstants.RQD_PREPEND_TIMESTAMP)
```

Comment:
This section seems to be ignoring `stderr` altogether.

Reply:
no, stderr is piped into stdout at line 401 (only in Windows and MacOS)